### ISSUE TITLE
Fcram firms don't boot and safe mode doesn't work

### ISSUE THREAD

**Issue by Alexyo21** [d0k3/GodMode9#774]
Good evening, I installed godmode9 as bootloader in both firm. I know there is  fastboot3ds, and it's recommended. But I appreciate godmode9 bootloader mode so I'll write it anyway. 
The problem is that when you try to boot to safe_firm through buttons combination, the 3ds will remain on black screen. And when you launch a firm like open_agb.firm through fcram boot, using its application on home menu , open_agb_launcher, the 3ds will remain on black screen.
Hope it'll be useful. Sorry for bothering you.
However I want to say that you have done and are you doing a great job. Keep doing it.
Godmode9 it's an awesome firm/app/file manager, bootloader (it's too much thing at the same time😅)
Thank you again

**Comment by d0k3** [comment]
Thanks for all the praise! Well, do you get the same issue when you boot other firms throgh the fcram boot? For example, Luma 3DS?

**Comment by Alexyo21** [comment]
I've installed it as bootloader, but yeah it can boot lumacfw from sd card by coldboot, from fcram I didn't tried lumacfw, as I know I tested just these two, because I made a quick test, I don't know very well but I can try again, as I get a little free time.

**Comment by Alexyo21** [comment]
> I've installed it as bootloader, but yeah it can boot lumacfw from sd card by coldboot, from fcram I didn't tried lumacfw, as I know I tested just these two, because I made a quick test, I don't know very well but I can try again, as I get a little free time.

**Comment by Alexyo21** [comment]
> Thanks for all the praise! Well, do you get the same issue when you boot other firms throgh the fcram boot? For example, Luma 3DS?

**Comment by LiskoFINAL** [comment]
Hi, I can add that fcram boot in latest godmode9 saltmode seems completely broken. OpenAGB launcher works fine in fastboot3ds with fcram boot enabled but with gm9 it locks in a black screen. I hope you'll watch after this because I really like gm9 salt as bootloader way more than fastboot3ds which is also very outdated and useless compared to gm9. Just as a temporary workaround could someone please provide me an updated to the last commit fastboot3ds firm? Thank you very much for this fantastic and definitive piece of software for the 3ds.

**Comment by Alexyo21** [comment]
> Hi, I can add that fcram boot in latest godmode9 saltmode seems completely broken. OpenAGB launcher works fine in fastboot3ds with fcram boot enabled but with gm9 it locks in a black screen. I hope you'll watch after this because I really like gm9 salt as bootloader way more than fastboot3ds which is also very outdated and useless compared to gm9. Thank you very much for this fantastic and definitive piece of software for the 3ds.




Yeah it's same problem I was talking about, taking about of godmode9 you're right about it's way more featured. But also fastboot3ds makes its work without problem. But in the end as you also I'd rather godmode9 as BL.

**Comment by d0k3** [comment]
Well, the bootloader in fb3DS is far superior to the one in GM9, as such stuff may work there and not in GM9. fb3DS is actually also designed as a bootloader (small feature set and super thorough testing for bugs). GM9 is designed as a complete package for everything a swiss army knife with a big feature set. I myself use fb3DS as bootloader.

Is there any specific reason why you guys want to stay with GM9 as bootloader? You know fb3DS is highly configurable, it stays in the background if you tell it to, and you can still configure a button shortcut for loading GM9.

**Comment by Alexyo21** [comment]
I know about it, fastboot has other advantages over gm9 and my other new Nintendo 3ds I use it, and I actually like it, but it doesn't matter the main purpose of my post was advise you about it, only make you know. I didn't mean to offend about your GitHub projects ( by the way both fantastic) or other not so good things.

**Comment by d0k3** [comment]
Yeah, and it's good this bug is here, for documentation. I just don't see this being fixed anytime soon. Bootloader stuff is complicated, and bootloader mode really is just still in there cause people want it to stay.

**Comment by Alexyo21** [comment]
> Yeah, and it's good this bug is here, for documentation. I just don't see this being fixed anytime soon. Bootloader stuff is complicated, and bootloader mode really is just still in there cause people want it to stay.

Yeah, I know it's not a ten minutes thing, so take your time. Exactly I reported for documentation in the end. 
I'd like to thank you for continuing to still carry updates on a old console like this.
